src/main/java/com/googlecode/jsendnsca/encryption/XorEncryptor.java

```diff
@@ -39,14 +39,18 @@ public void encrypt(byte[] passiveCheckBytes, byte[] initVector, String password
         }
 
         if (isNotBlank(password)) {
-            final byte[] passwordBytes = password.getBytes();
+            try {
+                final byte[] passwordBytes = password.getBytes("US-ASCII");
 
-            for (int y = 0, x = 0; y < passiveCheckBytes.length; y++, x++) {
-                if (x >= passwordBytes.length) {
-                    x = 0;
+                for (int y = 0, x = 0; y < passiveCheckBytes.length; y++, x++) {
+                    if (x >= passwordBytes.length) {
+                        x = 0;
+                    }
+                    passiveCheckBytes[y] ^= passwordBytes[x];
                 }
-                passiveCheckBytes[y] ^= passwordBytes[x];
+            } catch (Exception e) {
```

Comment:
AFAICS, there was no checked exceptions being thrown so is there any need to catch runtime exceptions and throw them wrapped in RuntimeException?

Reply:
According to javadoc getBytes(String) method throws an UnsupportedEncodingException :

public byte[] getBytes(String charsetName)
                throws UnsupportedEncodingException

I've used the same pattern as in BlowfishEncryptor and AESEncryptor